kbase-extension/static/kbase/js/common/jobManager.js

```diff
@@ -179,28 +179,32 @@ define(['common/jobMessages', 'common/jobs'], (JobMessages, Jobs) => {
                 jobIdList = [jobIdList];
             }
 
-            jobIdList.forEach((jobId) => {
-                if (!this.listeners[jobId]) {
-                    this.listeners[jobId] = {};
-                }
+            jobIdList
+                .filter((jobId) => {
+                    return jobId && jobId.length > 0 ? 1 : 0;
+                })
```

Comment:
some naughty undefined and null values were getting through. This'll stop them in their tracks!

Reply:
🛑

Reply:
With filter, you should just be able to return the boolean. I don't think you need the ternary.